sdks/java/extensions/sql/datacatalog/build.gradle

```diff
@@ -26,7 +26,8 @@ dependencies {
   compile library.java.grpc_google_cloud_datacatalog_v1beta1
   compile library.java.proto_google_cloud_datacatalog_v1beta1
   provided project(":sdks:java:extensions:sql")
-
+  provided project(":sdks:java:extensions:sql:zetasql")
```

Comment:
This should be `testRuntime` since it is not required by the main code.

Reply:
This can be a future improvement.